File: PRAK-6/PRAK601-2410817110011-MUHAMMADAZRIELAKBAR.py
baris, kolom = map(int, input().split())
elemen = list(map(int, input().split()))
matriks = []
indeks = 0
for a in range(baris):
    angka = elemen[indeks:indeks + kolom]
    matriks.append(angka)
    indeks += kolom
print()
for a in range(baris):
    for b in range(kolom):
        print(matriks[a][b], end=" ")
    print()

File: PRAK-6/PRAK601-2410817110011-MUHAMMADAZRIELAKBAR.c
#include <stdio.h>
int main()
{
  int baris, kolom;
  printf("");
  scanf("%d %d", &baris, &kolom);

  int matriks[baris][kolom];
  for (int a = 0; a < baris; a++)
  {
    for (int b = 0; b < kolom; b++)
    {
      scanf("%d", &matriks[a][b]);
      printf("%d ", matriks[a][b]);
    }
    printf("\n");
  }
}
